packages/web-react/src/components/FieldGroup/README.md

```diff
@@ -94,18 +94,19 @@ Validation states can be presented either by adding the `validationState` attrib
 
 ## API
 
-| Name              | Type                                           | Default | Required | Description                              |
-| ----------------- | ---------------------------------------------- | ------- | -------- | ---------------------------------------- |
-| `form`            | `string`                                       | `null`  | ✕        | Parent form ID                           |
-| `helperText`      | `string`                                       | `null`  | ✕        | Custom helper text                       |
-| `id`              | `string`                                       | —       | ✔        | Group and label identification           |
-| `isDisabled`      | `bool`                                         | `false` | ✕        | If true, the group is disabled           |
-| `isLabelHidden`   | `bool`                                         | `false` | ✕        | If true, label is hidden                 |
-| `isRequired`      | `bool`                                         | `false` | ✕        | If true, the group is marked as required |
-| `label`           | `string`                                       | —       | ✔        | Label text                               |
-| `name`            | `string`                                       | `null`  | ✕        | Group name                               |
-| `validationState` | [Validation dictionary][dictionary-validation] | `null`  | ✕        | Type of validation state                 |
-| `validationText`  | [`string` \| `string[]`]                       | `null`  | ✕        | Validation text                          |
+| Name              | Type                                           | Default | Required | Description                                                |
+| ----------------- | ---------------------------------------------- | ------- | -------- | ---------------------------------------------------------- |
+| `form`            | `string`                                       | `null`  | ✕        | Parent form ID                                             |
+| `helperText`      | `string`                                       | `null`  | ✕        | Custom helper text                                         |
+| `id`              | `string`                                       | —       | ✔        | Group and label identification                             |
+| `isDisabled`      | `bool`                                         | `false` | ✕        | If true, the group is disabled                             |
+| `isFluid`         | `bool`                                         | —       | ✕        | If true, the element spans to the full width of its parent |
```

Comment:
```suggestion
| `isFluid`         | `bool`                                         | `false`       | ✕        | If true, the element spans to the full width of its parent |
```

Reply:
I forgot. 🙈

Reply:
It's valid default value, because it's false when it's not set. We are in React. 🙈

Reply:
This is `undefined`, thus `false` by default.